dagger-common/src/main/java/io/odpf/dagger/common/core/Constants.java

```diff
@@ -5,10 +5,8 @@
     public static final boolean SCHEMA_REGISTRY_STENCIL_ENABLE_DEFAULT = false;
     public static final String SCHEMA_REGISTRY_STENCIL_URLS_KEY = "SCHEMA_REGISTRY_STENCIL_URLS";
     public static final String SCHEMA_REGISTRY_STENCIL_URLS_DEFAULT = "";
-    public static final String SCHEMA_REGISTRY_STENCIL_REFRESH_CACHE_KEY = "SCHEMA_REGISTRY_STENCIL_REFRESH_CACHE";
```

Comment:
lets keep this config? we can always have the option

Reply:
Since dagger uses absolute versioned URL, which doesn't change schema even if refresh. Thought we can remove this altogether.

Reply:
yes but some users(apart from gojek) might find it useful in my opinion